README.md

```diff
@@ -50,3 +50,26 @@ visualize('MyComponent', () => {
     xsnap('ignore this test', <MyComponent/>);
 });
 ```
+
+## Testing asynchronious actions
+
+In order for the asynchronious tests to work, `eyes-storybook` must be configured in a way it knows to wait for the component to notify it is done. This is done by setting a flag on the relevant story and inform applitools to look for this flag and wait.
+
+`storybook-snapper` provides a preconfigured `applitools.config.js` which you should use in your own `applitools.config.js` file.
+
+Change your `applitools.config.js` file to the following:
+```js
+const appName = require('./package').name;
+const applitoolsConfig = require('storybook-snapper/config/applitools.config');
+
+// optional local configuration file for overrides
+let config;
+
+try {
+  // for local testing you can add the `apiKey` to your private configuration file  
+  config = require('./applitools.private.config.js');
```

Comment:
not related to async testing

Reply:
No, but if we leave the config file as we talked about earlier, then we might want to document it, no?

Reply:
so document it correctly. currently, it is sitting under the "async testing" topic which is not related.

Reply:
Oh, I see.
Cool 👍

Reply:
Wait, actually, it **is** related to async testing.
The only reason we need to provide a configuration file is to have async tests.
We offer a preconfigured configuration file in order for the async tests to work.

Reply:
so it sounds to me like a general prerequisite. something that should come right after the installation part